### ISSUE TITLE
Random duplication of authorisation header parameter

### ISSUE THREAD

**Issue by vasiliqa** [rswag/rswag#110]
In my swagger_helper
```
      securityDefinitions: {
        token: {
          type: :apiKey,
          name: 'Authorization',
          in: :header
        }
      },
      security: [token: {}]
```
For some reason the `Authorization` parameter is repeated twice, and sometimes three times in the generated json
 ```
 "/sendings/{sending_id}/links": {
      "get": {
        "parameters": [
          {
            "name": "sending_id",
            "in": "path",
            "type": "integer",
            "required": true
          },
          {
            "name": "Authorization",
            "in": "header",
            "type": "string",
            "required": true
          },
          {
            "name": "Authorization",
            "in": "header",
            "type": "string",
            "required": true
          },
          {
            "name": "Authorization",
            "in": "header",
            "type": "string",
            "required": true
          }
        ],
        "responses": {
          "200": {
            "description": "Links list",
            "schema": {
              "type": "array",
              "items": {
                "$ref": "#/definitions/link"
              }
            },
          },
          "404": {
            "description": "Sending not found",
            "schema": {
              "$ref": "#/definitions/errors_object"
            }
          }
        }
      }
    }
```
What I'm doing wrong?

**Comment by domaindrivendev** [comment]
Can you provide the your respect for that operation?

**Comment by domaindrivendev** [comment]
Sorry typo - can you provide the rspec

**Comment by vasiliqa** [comment]
```
require 'swagger_helper'

RSpec.describe LinksController do
  let(:own_sending) { sendings(:one) }
  let(:foreign_sending) { sendings(:two) }
  let(:current_user) { users(:one) }
  let(:Authorization) { "Bearer #{ current_user.api_token }" }

  path '/sendings/{sending_id}/links' do
    get 'Find all links' do
      parameter name: :sending_id, in: :path, type: :integer, description: 'ID of links sending'

      response '200', 'Links list' do
        let(:sending_id) { own_sending.id }

        schema type: 'array',
               items: { '$ref' => '#/definitions/link' }

        run_test!
      end

      response '404', 'Sending not found' do
        let(:sending_id) { foreign_sending.id }

        schema '$ref' => '#/definitions/errors_object'

        run_test!
      end
    end
  end
end
```

**Comment by domaindrivendev** [comment]
The auth parameter isnt even defined in that spec, very odd. Could you also show your swagger_helper.rb?

**Comment by domaindrivendev** [comment]
Oh wait, I see it above

**Comment by akurashev** [comment]
Fixed in #112
